### PR TITLE
Feature/no charger element

### PR DESCRIPTION
**Pull request by Kafr15**
No charger.   Pertaining to Flare, but also quite general

### CODE REVIEW

File: ebmlite/schemata/mide_manifest.xml [hunk 1]
```diff
@@ -154,6 +154,8 @@
         <!-- Battery charger ICs -->
         <MasterElement name="ChargerMCP73837" id="0x4D60" mandatory="0" multiple="0" minver="1">Indicates the presence of a MCP73837 battery charger.</MasterElement>
         <MasterElement name="ChargerMAX14747" id="0x4D61" mandatory="0" multiple="0" minver="1">Indicates the presence of a MAX14747 battery charger.</MasterElement>
+        <MasterElement name="NoCharger" id="0x4D61" mandatory="0" multiple="0" minver="1">Indicates the presence of no battery charger or just a single cell battery.</MasterElement>
+
```
Comment (pscheidler) [inline ebmlite/schemata/mide_manifest.xml:158]: Big issue, we can't have 2 elements with the same ID, so can you change the NoCharger to maybe 0x4D62? Please make sure that ID is not used elsewhere
Smaller issue, can you change the name to match the style of similar elements. Maybe change it to "ChargerNone"

**Review by pscheidler — CHANGES_REQUESTED** [review]
Thanks for getting this in, but can't duplicate IDs

**Review by StokesMIDE — CHANGES_REQUESTED** [review]
Did this review earlier but forgot to submit it. Same comment as Pete, re: duplicate ID.

**Comment by Kafr15** [comment]
Ah, it was supposed to be 4D62 but it never got saved.  Amateur mistake.  I'll change it to 4D6F like David mentioned just to be safe though

**Review by StokesMIDE — APPROVED** [review]
Looks good.

**Pull request merged**